installer/templates/new/widgets/text/text.js

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
 import Widget from '../../assets/javascripts/widget';
+import {updatedAt} from '../../assets/javascripts/helpers';
 
 import './text.scss';
 
 Widget.mount(class Text extends Widget {
   render() {
+    let status = "";
+    if (this.state.widgetStatus) {
+      status = this.state.widgetStatus;
+    }
     return (
-      <div className={this.props.className}>
-        <h1 className="title">{this.props.title}</h1>
+      <div className={`${this.props.className} ${status}`}>    
```

Comment:
I propose we just pass the status as a simple string from the jobs and apply them to a style in the widget to decouple the job from layout. Yes?